internal/config/options.go

```diff
@@ -219,8 +219,7 @@ func WithLabelFilters(labelFilters []string) Option {
 
 func WithAnnotationFilters(annotationFilters []string) Option {
 	return func(o *options) {
-
-		filters := []string{}
+		filters := o.annotationsFilter
```

Comment:
Why change this? Do we not want `WithAnnotationFilters` to replace existing filters?

Reply:
this is so that I can use the default filter above, otherwise it [will always be overridden](https://github.com/open-telemetry/opentelemetry-operator/blob/main/main.go#L229)

Reply:
but also, I kind of expect these to be additive

Reply:
Ok, fair enough. Can you document that this function is additive, then?